content/en/real_user_monitoring/session_replay/_index.md

```diff
@@ -65,16 +65,21 @@ if (user.isAuthenticated) {
 
 To stop the Session Replay recording, call `stopSessionReplayRecording()`.
 
-### Disable Session Replay
+## Disable Session Replay
 
-To stop session recordings, remove `startSessionReplayRecording()` and set `sessionReplaySampleRate` to `0`. This stops collecting data for [Browser RUM & Session Replay plan][5], which includes replays.
+To stop session recordings, remove `startSessionReplayRecording()` and set `sessionReplaySampleRate` to `0`. This stops collecting data for [Browser RUM & Session Replay plan][2], which includes replays.
 
+[1]: /real_user_monitoring/browser/#setup
+[2]: https://www.datadoghq.com/pricing/?product=real-user-monitoring--session-replay#real-user-monitoring--session-replay
```

Comment:
These two links are out of place and are messing up the first two links on the page. These should be numbered 6 and 7, and the links in lines 40 and 70 pointed to them instead.